### ISSUE TITLE
Response code 401 (Unauthorized)

### ISSUE THREAD

**Issue by Holinc19** [vishwanatharondekar/gitlab-cli#207]
Couldn't create merge request
Response code 401 (Unauthorized)

**Comment by RoyRao2333** [comment]
@Holinc19 Same here. 

Did't try 2.0.9 but you can use 2.0.7 which works for me.

Still, hoping to get a fix of latest version.

**Comment by vishwanatharondekar** [comment]
Didnt know this was still being used given there are some close alternatives available. I will put some time into this to check this issue and update.

**Comment by Holinc19** [comment]
> Didnt know this was still being used given there are some close alternatives available. I will put some time into this to check this issue and update.


It works for me adding host field in options variables. I guess there is something wrong in `require('@gitbeaker/node')`

```
    var options = {
      host: git.config.get('gitlab.url') || process.env.GITLAB_URL,
      url: git.config.get('gitlab.url') || process.env.GITLAB_URL,
      token: git.config.get('gitlab.token') || process.env.GITLAB_TOKEN,
    };
```

**Comment by Holinc19** [comment]
> > Didnt know this was still being used given there are some close alternatives available. I will put some time into this to check this issue and update.
> 
> It works for me adding host field in options variables. I guess there is something wrong in `require('@gitbeaker/node')`
> 
> ```
>     var options = {
>       host: git.config.get('gitlab.url') || process.env.GITLAB_URL,
>       url: git.config.get('gitlab.url') || process.env.GITLAB_URL,
>       token: git.config.get('gitlab.token') || process.env.GITLAB_TOKEN,
>     };
> ```


The host would be set to `https://gitlab.com` without host options in @gitbeaker\requester-utils\dist\index.js.

```
        var _b = _a === void 0 ? {} : _a, token = _b.token, jobToken = _b.jobToken, oauthToken = _b.oauthToken, sudo = _b.sudo, profileToken = _b.profileToken, requesterFn = _b.requesterFn, _c = _b.profileMode, profileMode = _c === void 0 ? 'execution' : _c, _d = _b.host, host = _d === void 0 ? 'https://gitlab.com' : _d, _e = _b.prefixUrl, prefixUrl = _e === void 0 ? '' : _e, _f = _b.version, version = _f === void 0 ? 4 : _f, _g = _b.camelize, camelize = _g === void 0 ? false : _g, _h = _b.rejectUnauthorized, rejectUnauthorized = _h === void 0 ? true : _h, _j = _b.requestTimeout, requestTimeout = _j === void 0 ? 300000 : _j;
        if (!requesterFn)
            throw new ReferenceError('requesterFn must be passed');
```

**Comment by vishwanatharondekar** [comment]
Thanks for raising this @Holinc19 @RoyRao2333 . I couldnt wrap my head around what was causing this since it was working for me and @RoyRao2333 mentioned it working earlier. On investigation found out that I changed the gitlab package in 2.0.10 which expects `host` instead of `url`. Since I was testing on gitlab.com public repos it worked in my testing.

I am fine with merging https://github.com/vishwanatharondekar/gitlab-cli/pull/210 just need one small change. Remove the url getting set since thats not required anymore.

**Comment by vishwanatharondekar** [comment]
Merged #210 and published 2.0.12 to fix this.
